### ISSUE TITLE
[Bug]: SCP: Secret Laboratory Exiled server won't start

### ISSUE THREAD

**Issue by ThijmenNLD** [QuintenQVD0/Q_eggs#46]
### Panel Version

1.11.3

### Wings Version

1.11.5

### Service

SCP: Secret Laboratory Exiled

### Modified

No, I did not modify the egg

### Expected Behavior

When I press run the server will start without any issues

### Actual Behavior

When I press run the server starts but the server kills the game process after some time.
![2340604823056730](https://user-images.githubusercontent.com/73742011/231491474-94275e69-79b3-4c98-b593-a634e8bf6efd.jpg)

### Steps To Reproduce

Install the server
Server automatically pulls the new updates
Run the server
Get the problem

### Install logs

https://pteropaste.com/clexcxxp51im

**Comment by QuintenQVD0** [comment]
Does is woek withoud mods / plugins?

**Comment by ThijmenNLD** [comment]
No, it doesn't work even if I have no mods / plugins installed.

**Comment by ThijmenNLD** [comment]
Also it is an Oracle Cloud Instance running Ubuntu 22.04.

**Comment by QuintenQVD0** [comment]
It did work fine for me.

**Comment by QuintenQVD0** [comment]
Did the host os run out of resources?

**Comment by ThijmenNLD** [comment]
Is there any way to see if it ran out of resources?

**Comment by ThijmenNLD** [comment]
> Did the host os run out of resources?

No I don't think the server ran out of resources.

**Comment by ThijmenNLD** [comment]
I can try using another Ubuntu version.

**Comment by QuintenQVD0** [comment]
You can try, but that should not matther

**Comment by ThijmenNLD** [comment]
I accidentally deleted my OCI so I created a new one running Ubuntu 20.04. I ran the server but still encounter the same issue.

**Comment by ThijmenNLD** [comment]
Are there any other troubleshooting methods I can try?

**Comment by ThijmenNLD** [comment]
Added the 2 exports to the startup and sadly still get the error.

**Comment by QuintenQVD0** [comment]
> Added the 2 exports to the startup and sadly still get the error.

it should print out way more logs. you have them?

**Comment by ThijmenNLD** [comment]
Where can I locate the logs?
Also do I need to put this in the startup command: "export FEX_APP_DATA_LOCATION=/home/container/rootfs/; export FEX_APP_CONFIG_LOCATION=/home/container/; export XDG_DATA_HOME=/home/container; export HOME=/home/container; export FEX_SILENTLOG=0; export FEX_OUTPUTLOG=stderr; FEXInterpreter LocalAdmin {{SERVER_PORT}}"

**Comment by QuintenQVD0** [comment]
> Where can I locate the logs? Also do I need to put this in the startup command: "export FEX_APP_DATA_LOCATION=/home/container/rootfs/; export FEX_APP_CONFIG_LOCATION=/home/container/; export XDG_DATA_HOME=/home/container; export HOME=/home/container; export FEX_SILENTLOG=0; export FEX_OUTPUTLOG=stderr; FEXInterpreter LocalAdmin {{SERVER_PORT}}"

it should print them in the console. else in the rootfs folder or just in the home dir

**Comment by ThijmenNLD** [comment]
Where are the logs located if they are in the RootFS folder?

**Comment by ThijmenNLD** [comment]
Do I need to go to rootfs/RootFS/Ubuntu_20_04?

**Comment by QuintenQVD0** [comment]
> Do I need to go to rootfs/RootFS/Ubuntu_20_04?

no, it should just use the rootfs folder. does it print anything extra in the console?

**Comment by ThijmenNLD** [comment]
No, tho when I looked inside of rootfs/RootFS/Ubuntu_20_04 I found a file called: "Unknown_Packages" which contains this:
Package Not Found: libxcb-imdkit1
Package Not Found: libllvm13
Package Not Found: libllvm13:i386
Package Not Found: mangohud

Can I ignore this?

**Comment by QuintenQVD0** [comment]
> No, tho when I looked inside of rootfs/RootFS/Ubuntu_20_04 I found a file called: "Unknown_Packages" which contains this: Package Not Found: libxcb-imdkit1 Package Not Found: libllvm13 Package Not Found: libllvm13:i386 Package Not Found: mangohud
> 
> Can I ignore this?

that is fine

**Comment by QuintenQVD0** [comment]
I asked in there discord

**Comment by ThijmenNLD** [comment]
Also here is my Pterodactyl server console when starting it up:

https://user-images.githubusercontent.com/73742011/235142584-557b9888-4c3a-492f-aaa1-551754c06a66.mp4

**Comment by QuintenQVD0** [comment]
> Also here is my Pterodactyl server console when starting it up:
> 
>  PterodactylServerConsole.mp4

did you hit stop?

**Comment by ThijmenNLD** [comment]
Yes

**Comment by QuintenQVD0** [comment]
The segfault is normal as you hit stop before it fully started

**Comment by ThijmenNLD** [comment]
Is there anything I might have done wrong when installing the server that causes the problem?

**Comment by QuintenQVD0** [comment]
Not realy. I am working with the dev of an other emulator to get them to support it.

**Comment by ThijmenNLD** [comment]
May I ask what the name of the other emulator is?

**Comment by QuintenQVD0** [comment]
> 

Ofc, that is just box64

**Comment by ThijmenNLD** [comment]
Maybe a bit rude to ask, but are there any updates?

**Comment by QuintenQVD0** [comment]
The progress to make the box64 emulator work are not ik my hands so you will need to wait for the dev.

**Comment by ThijmenNLD** [comment]
Your OCI runs the server fine right?

**Comment by QuintenQVD0** [comment]
> Your OCI runs the server fine right?

it did, It has been a wile since I started the server

**Comment by ThijmenNLD** [comment]
For some reason, the SCP:SL server does start now and seems to work, though I'm encountering some issues, like certain game mechanics not fully functioning. For example, as a certain SCP, you can revive people, but it doesn't work. As an SCP, you can teleport every 3 seconds; it still works, but the time is not displayed as it normally would be. Certain commands also don't work (built-in commands and plugin commands). I think this has to do with the poor emulation of the emulator, but I'm not sure though.

**Comment by QuintenQVD0** [comment]
> For some reason, the SCP:SL server does start now and seems to work, though I'm encountering some issues, like certain game mechanics not fully functioning. For example, as a certain SCP, you can revive people, but it doesn't work. As an SCP, you can teleport every 3 seconds; it still works, but the time is not displayed as it normally would be. Certain commands also don't work (built-in commands and plugin commands). I think this has to do with the poor emulation of the emulator, but I'm not sure though.

I requested some extra packages for fex and they added them. So yea now libssl 1.1 is in the rootfs. The emulator tryes its best notthing more it can realy do as those are manly made for running the client and not the server.
